lib/src/collect.dart

```diff
@@ -97,6 +97,8 @@ Future<Map<String, dynamic>> _getAllCoverage(
   scopedOutput ??= <String>{};
   final vm = await service.getVM();
   final allCoverage = <Map<String, dynamic>>[];
+  final ver = await service.getVersion();
+  final reportLines = ver.major == 3 && ver.minor != null && ver.minor! >= 51;
```

Comment:
Do we intend for this to flip off when the VM service bumps to `4.0.0` or should we add `version.major > 3 ||`?

Reply:
Done